torch/csrc/lazy/ts_backend/ts_node.cpp

```diff
@@ -98,10 +99,11 @@ ShapeCache* GetShapeCache() {
 
 Shape TsNode::GetOpShape(
     const std::function<Shape()>& shape_fn) const {
+	auto hash = getenv("SHAPE_HASH_FIX") ? hash_with_sizes() : hash_without_sizes(); // TODO will clean up this line before land
```

Comment:
what is this line for actually?  and when you land what would you change it to?

Reply:
The line is here so that I can turn on/off the fix. This way I can confirm it really fixed the issue we encountered in the bert model: avoid the case that the model seems working because the issue is not actually reproduced.

I've confirmed the fix actually. So I'll change it to
```
auto hash = hash_with_sizes();
```

now